subsys/bluetooth/mesh/net.c

```diff
@@ -98,8 +96,16 @@ struct bt_mesh_net bt_mesh = {
  */
 static bool ivi_was_recovered;
 
-NET_BUF_POOL_DEFINE(loopback_buf_pool, CONFIG_BT_MESH_LOOPBACK_BUFS,
-		    LOOPBACK_MAX_PDU_LEN, LOOPBACK_USER_DATA_SIZE, NULL);
+struct loopback_buf {
+	sys_snode_t node;
+	struct bt_mesh_subnet *sub;
+	uint8_t len;
+	uint8_t data[];
```

Comment:
```suggestion
	uint8_t data[LOOPBACK_MAX_PDU_LEN];
```
to ensure that the slab definition and the buffer lengths are consistent